README.md

```diff
@@ -114,3 +114,26 @@ Not every network's explorer supports multi-file verification.
 For those networks, the corresponding compiler plugin's `flatten` functionality is invoked, in order to verify the contract as a single file.
 
 **NOTE**: You must set an Etherscan API key environment variable to use the publishing feature.
+
+## Custom Networks
+
+If you would like to use ape-etherscan with your [custom network configuration](https://docs.apeworx.io/ape/stable/userguides/networks.html#custom-network-connection), you can use the same network identifier you used to configure it.  For instance, with a custom Ethereum network called "apechain" your configuration might look something like this:
```

Comment:
nit: we have an unwritten markdown style rule of 1 sentence per line.
it started in a github comment way back in the day but ever since then, all the markdown, we have done that.

it renders to the same

Reply:
Sure, no problems matching style.  Will update.

But also, why?  Doesn't that make reading them raw pretty bad?  One of the benefits of markdown is that it's nice to read as text as well.

Reply:
The only reason is consistency, but I dont think one sentence per line is at all bad to read.

Reply:
Not why should I make the change, but why was it originally decided to do that?

Reply:
I don't remember and the only thing i really care about is consistency.
We can potentially change the style throughout all the docs.

Reply:
Wasn't arguing or making a suggestion.  I was just curious.